docs/tutorials/configure-app.md

```diff
@@ -59,12 +59,12 @@ window.ENV = 'dev';
   "home": {
     "headerSettings": {
       "backgroundColor": "#FF9800",
-      "backgroundImage": "/images/backgrounds/home.png",
+      "backgroundImage": "/images/backgrounds/galaxy-bg.png",
```

Comment:
The image looks very nice 😊 two things:

* Did you check that it looks good with different screen resolutions?
* The file is nearly 1MB in size, which is quite large. Can you minify it without visible quality loss?

Reply:
a) Yes, but I'd want to discuss with you largening the header image anyways, as this is now is causing responsive issues on some devices. 
b) I already minified it to the max.